app/Http/helpers.php

```diff
@@ -855,3 +855,56 @@ function isRtlForLang()
 {
     return in_array(app()->getLocale(), ['ar', 'he']);
 }
+
+/**
+ * Deletes all user sessions except the current session.
+ *
+ * This function checks the session driver and deletes sessions based on the user ID.
+ * For file-based sessions, it reads session files and deletes those that match the user ID.
+ * For other drivers (like database or Redis), it uses the logoutOtherDevices method.
+ *
+ * @param  int  $userId  The ID of the user whose sessions are to be deleted.
+ * @param  string  $password  The user's password for authentication.
+ * @return void
+ */
+function deleteUserSessions(int $userId, string $password): void
+{
+    if (config('session.driver') === 'file') {
```

Comment:
1. Don't use if else. Use early return for else statement.
2. So ifs and loops, You can use collect, each, reject with collections to do this

Suggestion: if you want to do something very advance there is a thing called laravel macro you can add a function to laravel \File facade to clean directory and ignore some files while doing it